incubator/starter/stack.yaml

```diff
@@ -1,6 +1,6 @@
 name: Starter Sample
 version: 0.1.0
-description: Runnable starter stack, copy to create a stack
+description: Runnable starter stack, copy to create a new stack
```

Comment:
Do we need "copy to create a new stack" in this description? 

The reason I'm asking is because you can with `appsody stack create --copy <stackname>` very easily now copy any other stack to create a new stack. Hence I'm worried that users might think that they have to start by copying this stack whereas don't we now have three starting points for creating stacks? This one, which is Starter Sample (Runnable), or the Sample (Template) stack, or copying another more complex existing stack.

With that in mind wouldn't "Runnable starter stack" be sufficient for this stack description?

Reply:
How about “Runnable stater stack - a basis for new stacks”?

Reply:
That sounds better to me - @neeraj-laad what are your thoughts?

Reply:
Sounds good to me

Reply:
We just have limited retail space on the website today for this and want to make sure that fits well in that limit.